app/templates/home.html

```diff
@@ -1,6 +1,12 @@
-<section class="navbar">
-  <!-- the ngRepeat directive  -->
-  <div ng-repeat="room in home.rooms">
-    {{room.$value}}
+<!-- the ngRepeat directive instantiates a template once per item from a collection. Each template instance gets its own scope, where the given loop variable is set to the current collection item, and $index is set to the item index or key -->
+<div class="menu">
+  <button class="create-btn" ng-click="home.showme=true" ng-hide="home.showme">Create Chat Room</button>
+  <form ng-show="home.showme">Enter a Room Name:
+    <input type="text" placeholder="Enter name here" ng-model="roomName">
+    <input type="submit" ng-click="home.showme=false; home.createRoom(roomName)" value="Create"></button>
```

Comment:
You already set `showme` to `false` in the `home.createRoom` method so you don't need it here. Doing it in the controller is cleaner.